test/src/io/github/teamfractal/entity/MarketTest.java

```diff
@@ -103,6 +97,48 @@ public void marketCanCheckResourceMoreThanAmountYouWantToBuy() {
 		assertFalse(market.hasEnoughResources(ResourceType.FOOD, 1000000));
 	}
 
+	/**
+	 * Made by JBT
```

Comment:
put the Made by JBT in their own comment above the docstring as theyre not documentation.
// Made by JBT
/**
 * docs here
 */
test here